common/src/main/java/com/wynntils/services/mapdata/features/impl/MapAreaImpl.java

```diff
@@ -62,12 +61,14 @@ public List<Location> getPolygonArea() {
 
     @Override
     public BoundingPolygon getBoundingPolygon() {
-        return boundingPolygon != null
-                ? boundingPolygon
-                : (boundingPolygon = BoundingPolygon.fromLocations(polygonArea));
+        return boundingPolygon != null ? boundingPolygon : (calculateBoundingPolygon());
```

Comment:
Are the () around `calculateBoundingPolygon()` needed for some kind of operator precedence? It looks weird.

Reply:
Hmm, probably not. My bad.